cmd/actuary/dockerhost_test.go

```diff
@@ -0,0 +1,352 @@
+// goal is for us to test all the functions that are being called
+//probably mock responses from the Docker API to not have to test with a real daemon
+
+package actuary
+
+
+import (
+	"os"
+	"path/filepath"
+	"fmt"
+	"encoding/json"
+	"testing"
+	"strconv"
+	"net/http" //Package http provides HTTP client and server implementations.
+	"net/http/httptest" //Package httptest provides utilities for HTTP testing.
+	"github.com/docker/engine-api/types"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/api"
+)
+
+//variables for tests, helper functions
+//Rest of test files use the following functions/variables
+//group together the api call and the expected object (in bytes)
+type callPairing struct{
+	call string
+	obj []byte
+}
+
+type imageList struct{
+	images []types.Image
+}
+
+type typeContainerList struct{
+	typeContainers []types.Container
+}
+
+//For testing functions that require a specific number of images
+func (list imageList) populateImageList(size int) (imageList){
+	list.images = nil
+	var img types.Image
+	for i := 0; i < size; i++{
+		img = types.Image{ID: strconv.Itoa(i)}
+		list.images = append(list.images, img)
+	}
+	return list
+}
+
+//For testing functions that require a specific number of containers
+func (list typeContainerList) populateContainerList(size int) (typeContainerList){
+	list.typeContainers = nil
+	var c types.Container
+	for i := 0; i < size; i++{
+		c = types.Container{ID: strconv.Itoa(i)}
+		list.typeContainers = append(list.typeContainers, c)
+	}
+	return list
+}
+
+var testTarget, err = NewTarget()
```

Comment:
Test target should not be global. We should create a `NewTestTarget()` method that returns a `Target` object that we test.